lib/archethic.ex

```diff
@@ -354,17 +354,21 @@ defmodule Archethic do
   Retrieve a transaction chain based on an address from the closest nodes
   by setting `paging_address as an offset address.
```

Comment:
Suggestion: you should probably change the doc here

Reply:
Done !